homeassistant/components/flexit_bacnet/sensor.py

```diff
@@ -0,0 +1,252 @@
+        state_class=SensorStateClass.MEASUREMENT,
+        # suggested_unit_of_measurement= UnitOfTime.MINUTES,
+        native_unit_of_measurement=UnitOfTime.MINUTES,
+        translation_key="rapid_ventilation_remaining_duration",
+        value_fn=lambda data: data.rapid_ventilation_remaining_duration,
+        suggested_display_precision=0,
+    ),
+    FlexitSensorEntityDescription(
+        key="supply_air_fan_control_signal",
+        state_class=SensorStateClass.MEASUREMENT,
+        translation_key="supply_air_fan_control_signal",
+        native_unit_of_measurement=PERCENTAGE,
+        # unit_of_measurement=PERCENTAGE,
+        value_fn=lambda data: data.supply_air_fan_control_signal,
+    ),
+    # What sensor type should be used for this sensor?
+    FlexitSensorEntityDescription(
+        key="supply_air_fan_rpm",
+        state_class=SensorStateClass.MEASUREMENT,
+        native_unit_of_measurement=REVOLUTIONS_PER_MINUTE,
+        translation_key="supply_air_fan_rpm",
+        value_fn=lambda data: data.supply_air_fan_rpm,
+    ),
+    FlexitSensorEntityDescription(
+        key="exhaust_air_fan_control_signal",
+        state_class=SensorStateClass.MEASUREMENT,
+        translation_key="exhaust_air_fan_control_signal",
+        value_fn=lambda data: data.exhaust_air_fan_control_signal,
+        native_unit_of_measurement=PERCENTAGE,
+    ),
+    FlexitSensorEntityDescription(
+        key="exhaust_air_fan_rpm",
+        state_class=SensorStateClass.MEASUREMENT,
+        native_unit_of_measurement=REVOLUTIONS_PER_MINUTE,
+        translation_key="exhaust_air_fan_rpm",
+        value_fn=lambda data: data.exhaust_air_fan_rpm,
+    ),
+    # This does not make sense to be a sensor, since it's value is always the same, i.e. 0.8 kW
+    # Could this be added as some form of extra attribute somewhere?
+    # FlexitSensorEntityDescription(
+    #     key="electric_heater_nominal_power",
+    #     device_class=SensorDeviceClass.POWER,
+    #     native_unit_of_measurement=UnitOfPower.KILO_WATT,
+    #     translation_key="electric_heater_nominal_power",
+    #     value_fn=lambda data: data.electric_heater_nominal_power,
+    #     suggested_display_precision=3,
+    # ),
+    FlexitSensorEntityDescription(
+        key="electric_heater_power",
+        device_class=SensorDeviceClass.POWER,
+        native_unit_of_measurement=UnitOfPower.KILO_WATT,
+        translation_key="electric_heater_power",
+        value_fn=lambda data: data.electric_heater_power,
+        suggested_display_precision=3,
+    ),
+    FlexitSensorEntityDescription(
+        key="air_filter_operating_time",
+        state_class=SensorStateClass.TOTAL_INCREASING,
+        suggested_display_precision=0,
+        suggested_unit_of_measurement=UnitOfTime.HOURS,
+        native_unit_of_measurement=UnitOfTime.HOURS,
+        translation_key="air_filter_operating_time",
+        value_fn=lambda data: data.air_filter_operating_time,
+        # should we have a last_reset?
```

Comment:
When does it reset?

Reply:
I think it automatically does that when you replace the filter in the unit.